functions/__z.fish

```diff
@@ -164,9 +164,14 @@ function __z -d "Jump to a recent directory."
         if set -q _flag_echo
             printf "%s\n" "$target"
         else if set -q _flag_directory
-            type -q xdg-open;and xdg-open "$target"; and return $status;
-            type -q open;and open "$target"; and return $status;
-            echo "Not sure how to open file manager"; and return 1;
+            if test "$OS" = Windows_NT
+                type -q explorer;and explorer "$target"; return $status;
+                # Be careful, in msys2, explorer always return 1
+            else
+                type -q xdg-open;and xdg-open "$target"; and return $status;
+                type -q open;and open "$target"; and return $status;
+                echo "Not sure how to open file manager"; and return 1;
```

Comment:
Let's move this line outside the if statement, so it also covers the case where `explorer` doesn't exist (although highly unlikely).

Reply:
It's done now.

Reply:
Apologies if I wasn't clear, what i meant was:

if `type -q explorer` fails, the `zo` command will currently do nothing at all, whereas I expect it to output `"Not sure how to open file manager"` and return 1.

This fallback is currently implemented only for unix, but we should make that fallback cover windows as well, so moving the fallback out of the if block would achieve that behaviour.